ui/lib/pki/addon/components/page/pki-issuer-list.hbs

```diff
@@ -41,7 +47,11 @@
   </:actions>
   <:list as |issuers|>
     {{#each issuers as |pkiIssuer idx|}}
-      <LinkedBlock class="list-item-row" @params={{array "issuers.issuer.details" pkiIssuer.id}} @linkPrefix={{@mountPoint}}>
+      <LinkedBlock
+        class="list-item-row"
+        @params={{array "issuers.issuer.details" @backend pkiIssuer.id}}
+        @linkPrefix={{@mountPoint}}
```

Comment:
is `@mountPoint` the static `pki` part of the route name? Just double checking it doesn't conflict with adding `@backend`

Reply:
That's right, it's the app's url to the PKI engine, so it is static